backend/src/modules/doubles-tournament/core/presentation/rest-api/DoublesTournamentRouter.ts

```diff
@@ -22,8 +27,27 @@ export function doublesTournamentRouter(
     return response.status(StatusCodes.OK).json(new TournamentTeamListDto(toTournamentTeamDto(queryResult)));
   };
 
+  const getMatchesQueueByTournamentId = async (request: Request, response: Response) => {
+    const { tournamentId } = request.params;
+    const queryResult = await queryPublisher.execute<FindMatchesQueueByTournamentByIdResult>(
+      new FindMatchesQueueByTournamentById({ tournamentId }),
+    );
+    if (!queryResult) {
+      return response
+        .status(StatusCodes.NOT_FOUND)
+        .json({ message: `Such Matches queue doesn't exist because doubles tournament with id = ${tournamentId} is not found!` });
+    }
+    const matchesQueueWithPrimitiveTypes = queryResult.queuedMatches.map((match) => ({
```

Comment:
Można to nazwać queuedMatchesDto i zrobić typ dla obiektu wewnątrz QueueMatchDto :) To łatwo to tez opiszesz w OpenAPI.

Reply:
https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/pull/52/commits/d877aafb3c727637de70143ceccca794d87f5dbe
nie robiłem całej kolejki jako Dto, ale obiekt meczu jedynie :-), bo wtedy mi źle typ zmiennej łapało tutaj
`const queuedMatchesDto: QueuedMatchDto[]`